posthog/hogql/test/test_placeholders.py

```diff
@@ -80,3 +81,127 @@ def test_replace_placeholders_with_cte(self):
         assert expr2.ctes is not None and expr2.ctes["test"] is not None
         assert isinstance(expr2.ctes["test"].expr, ast.SelectQuery)
         assert isinstance(expr2.ctes["test"].expr.select[0], ast.Constant)
+
+
+class TestBytecodePlaceholders(BaseTest):
+    """
+    These tests hit the _visit_placeholder_via_bytecode branch in
```

Comment:
I dont think `_visit_placeholder_via_bytecode` is a thing? Maybe an old comment before a refactor here?